javascript-source/handlers/subscribeHandler.js

```diff
@@ -20,17 +20,21 @@
  *
  * Register new subscribers and unsubscribers in the channel
  */
-(function() {
+(function () {
     var subMessage = $.getSetIniDbString('subscribeHandler', 'subscribeMessage', '(name) just subscribed!'),
         primeSubMessage = $.getSetIniDbString('subscribeHandler', 'primeSubscribeMessage', '(name) just subscribed with Twitch Prime!'),
         reSubMessage = $.getSetIniDbString('subscribeHandler', 'reSubscribeMessage', '(name) just subscribed for (months) months in a row!'),
         giftSubMessage = $.getSetIniDbString('subscribeHandler', 'giftSubMessage', '(name) just gifted (recipient) a subscription!'),
+        giftAnonSubMessage = $.getSetIniDbString('subscribeHandler', 'giftAnonSubMessage', '(name) just gifted (recipient) a subscription!'),
```

Comment:
Here I would make the message the same as Twitch's, so it consists with the other default ones.

"An anonymous viewer gifted (recipient) a subscription!"